tfjs-core/src/ops/conv2d_test.ts

```diff
@@ -228,6 +228,41 @@ describeWithFlags('conv2d', ALL_ENVS, () => {
     expectArraysClose(resultData, [20, 26, 13, 12]);
   });
 
+  it('x=[4,2,2] f=[2,2,4,4] s=1 d=1 p=same NCHW', async () => {
```

Comment:
I am not sure that it is reasonable that all channels of input are the same and are `[1 2 3 4]`, and all filters defined by output are the same and `[[3, 3, 3, 3], [1, 1, 1, 1], [5, 5, 5, 5], [0, 0, 0, 0]]`. Is it better to use some random number for inputs and filters.